### ISSUE TITLE
need a new release that works with python 3.8

### ISSUE THREAD

**Issue by dwalton76** [opencomputeproject/ocp-diag-pci_lmt#12]
Now that https://github.com/opencomputeproject/ocp-diag-pci_lmt/pull/10 is in, there should be a release so users can download a package with this fix so that they can run on python 3.8

**Comment by mimir-d** [comment]
@sksekar can you make a new release? thanks

**Comment by mimir-d** [comment]
@sksekar ping?

**Comment by sksekar** [comment]
Hi @mimir-d, I'm not sure what "make a new release" means. Is it just merging #10 into main? thanks

**Comment by mimir-d** [comment]
right, but there's no need anymore; #10 is in 1.4.0 and that's already on pypi

**Comment by sksekar** [comment]
Apologies @dwalton76 for the delay. My bad that it wasn't addressed at the right time.
